parsers/xmp.py

```diff
@@ -50,13 +49,14 @@ def items(self) -> List[SensorItem]:
         return []
 
     def format_version(self) -> Optional[str]:
-        return None
+        print("XMP format version", self)
 
     def serialize(self):
         """this method will write all the added items to file"""
-        pass
+        print("XMP serialize is not implemented", self)
```

Comment:
Usually an error is used: `raise NotImplementedError("XMP serialize is not implemented")`